app/models/vacancy.rb

```diff
@@ -67,13 +73,26 @@ class Vacancy < ApplicationRecord
   validate :enable_job_applications_cannot_be_changed_once_listed
 
   has_noticed_notifications
+  has_paper_trail on: [:update],
+                  only: ATTRIBUTES_TO_TRACK_IN_ACTIVITY_LOG,
+                  if: proc { |vacancy| vacancy.listed? }
```

Comment:
Do you maybe want `published?`?

I think `listed?` will only apply once it's gone live, so wouldn't cover changes made _after_ publishing but _before_ the date it's published for.

Not saying that `published?` is better, just worth thinking about/checking!

Reply:
Don't scheduled jobs also have the status `published`? The ticket only requires the activity log tab for active and closed jobs, so changes to scheduled jobs (before they're live but after they're 'published') won't be displayed anyway.

My goal here was to only show changes that have been made after the job goes live.